openassessment/xblock/test/test_leaderboard.py

```diff
@@ -128,17 +128,6 @@ def test_show_submissions_that_have_greater_than_0_score(self, xblock):
         ])
         self._assert_leaderboard_visible(xblock, True)
 
-    @scenario('data/leaderboard_show.xml')
```

Comment:
I'm removing this test. This was never a realistic test, and now it's breaking because the format it's using isn't recognized by the raw_answer parser. 

I ran the following query on the read-replica:
`select * from submissions_submission as sub join submissions_studentitem as si on sub.student_item_id = si.id where sub.raw_answer not like "{%" and si.item_type = 'openassessment';`
(get all openassessment submissions whose answers don't start with an opening bracket) and there are no such submissions in the table. every existing openassessment submission is a json dictionary.